fast64_internal/oot/oot_level_writer.py

```diff
@@ -130,8 +130,8 @@ def writeOtherSceneProperties(scene, exportInfo):
 	modifySceneTable(scene, exportInfo)
 	modifySegmentSymbols(scene, exportInfo)
 	modifySceneIDs(scene, exportInfo)
-	modifyDmaMgrFileNames(scene, exportInfo)
-	modifyDmaTableEntries(scene, exportInfo)
+	# modifyDmaMgrFileNames(scene, exportInfo)
```

Comment:
I'd say either remove this change or actually properly remove this feature rather than just commenting it out. I can make a pr later to do that so for now I'd say just revert these changes.

Reply:
Sure.